Azure.ResourceManager.Core/ResourceListOperations.cs

```diff
@@ -15,20 +15,20 @@ namespace Azure.ResourceManager.Core
     /// </summary>
     public class ResourceListOperations
     {
-        public static Pageable<TOperations> ListAtContext<TOperations, TResource>(
+        public static Pageable<ArmResourceOperations> ListAtContext<ArmResourceOperations, ArmResource>(
```

Comment:
I think we can remove the generic type completely
```suggestion
        public static Pageable<ArmResourceOperations> ListAtContext(
```

Reply:
If these methods are meant simply to call the generic ARM APIs, then the proper return type would be 'ArmResource', because it will have the ARM envelope as well as the operations, and then no generic parameters are needed at all, as Michael suggests.

I am assuming that this would then be used by generated list methods in the Collection classes to transform the ```Pageable<ArmResource>``` into ```Pageable<MyResourceOperations>``` when presenting a method to the user like ```Pageable<VirtualMachineOperations> List(ArmFilterCollection, int? CancellationToken)```